plugins/experimental/parent_select/strategy.h

```diff
@@ -36,7 +36,7 @@
 // TODO rename, move to respective sub-plugins
 #define PLUGIN_NAME "pparent_select"
 
-constexpr const char *PL_NH_DEBUG_TAG = "plugin_nexthop";
+constexpr const char *PL_NH_DEBUG_TAG = PLUGIN_NAME;
```

Comment:
Because of this change, you must also change `plugin_nexthop` in both places it is used in each of parent_select_peer.test.py and parent_select_peer2.test.py to `pparent_select`.

Reply:
Done